.eslintrc.js

```diff
@@ -1,4 +1,3 @@
 module.exports = {
-  extends: ['@mate-academy/eslint-config', 'plugin:cypress/recommended', 'plugin:chai-friendly/recommended'],
-  plugins: ['cypress', 'chai-friendly'],
```

Comment:
The 'plugin:chai-friendly/recommended' configuration is missing. Please add it back to the 'extends' array to ensure adherence to the recommended code style linting. 🧐